webrtc-stats.html

```diff
@@ -4209,6 +4218,331 @@ <h2>
           </section>
         </div>
       </section>
+      <section id="encoding-parameters-dict*">
+        <h3>
+          <dfn>RTCEncodingParameterStats</dfn> dictionary
+        </h3>
+        <div>
+          <pre class="idl">dictionary RTCEncodingParameterStats : RTCStats {
+             DOMString rid;
+             octet payloadType;
+             boolean active;
+             unsigned long ptime;
+             unsigned long maxBitrate;
+             double maxFramerate;
+             double scaleResolutionDownBy;
+             sequence&lt;DOMString&gt; scalabilityModes;
+  };</pre>
+          <section>
+            <h2>
+              Dictionary <a class="idlType">RTCEncodingParameterStats</a> Members
+            </h2>
+            <dl data-link-for="RTCEncodingParameterStats" data-dfn-for=
+            "RTCEncodingParameterStats" class="dictionary-members">
+              <dt>
+                <dfn><code>rid</code></dfn> of type <span class="idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>
+                  Restriction Identifier(rid) of the encoding.
```

Comment:
Can we reference the members of [RTCRtpEncodingParameters](https://w3c.github.io/webrtc-pc/#dom-rtcrtpencodingparameters) for all of these, to make it clear where we get this information from?